E3Next/Settings/CharacterSettings.cs

```diff
@@ -337,7 +342,17 @@ private void LoadData()
 
 
             LoadKeyData("Heals", "Tank", ParsedData, HealTankTargets);
+            for (Int32 i = 0; i < HealTankTargets.Count; i++)
+            {
+                HealTankTargets[i] = e3util.FirstCharToUpper(HealTankTargets[i]);
+            }
+
             LoadKeyData("Heals", "Important Bot", ParsedData, HealImportantBotTargets);
+            //upper case first letter on all important bots, netbots bug that doesn't like lower case.
+            for (Int32 i = 0; i < HealImportantBotTargets.Count; i++)
+            {
+                HealImportantBotTargets[i] = e3util.FirstCharToUpper(HealImportantBotTargets[i]);
```

Comment:
Same

Reply:
https://itecnote.com/tecnote/c-make-first-letter-of-a-string-upper-case-with-maximum-performance/

Reply:
TIL